demo/src/app/components/datepicker/overview/datepicker-overview.component.html

```diff
@@ -134,17 +134,17 @@ <h4>Handling the popup</h4>
   <h4>Adapters</h4>
 
   <p>
-    You can also tell datepicker to use the native javascript date adapter (bundled with ng-bootstrap) as in the
-    <a routerLink="../examples" fragment="adapter">custom date adapter example</a>. For now
-    the adapter works only for the form integration, so for instance <code>(ngModelChange)</code>
+    You can also tell datepicker to use the native javascript date adapter (bundled with ng-bootstrap).
```

Comment:
I believe there was an error in copied statement, and the `JavaScript` is a proper name, so character case should be corrected